modules/programs/fish.nix

```diff
@@ -12,6 +12,23 @@ let
     mapAttrsFlatten (k: v: "alias ${k} '${v}'") cfg.shellAliases
   );
 
+  babelfish = pkgs.callPackage ../../pkgs/shells/fish/plugins/babelfish.nix { };
```

Comment:
I don't really want to keep package definitions in this repo.

Since this isn't guaranteed to be available in nixpkgs (eg. 20.09) the best solution is to add a package option without a default, that way it's up to the user to either introduce the version they want into their configuration or if nixpkgs is recent enough just use `pkgs.babelfish`. The later can then also the default once the oldest supported release has babelfish available.

Reply:
I agree. It was convenient to spike it, but I wouldn't want to vendor an old package in nix-darwin. I'll add a package option.